src/main/java/org/example/Main.java

```diff
@@ -19,10 +21,31 @@ public static void main(String[] args) {
             out.println(input);
             if (CORRECT_NUMBER.equals(input)) {
                 out.println("Wszedles!");
+                menu();
                 return;
             }
             numberOfTries++;
             out.println("Nie prawidlowe haslo!\n");
         }
     }
+    public static void menu() {
```

Comment:
name of the method should be descriptive. menu what? print menu? delete menu? burger menu?